app/uk/gov/hmrc/thirdpartydeveloperfrontend/controllers/addapplication/AddApplication.scala

```diff
@@ -217,7 +217,7 @@ class AddApplication @Inject() (
         userEmailPreferences: EmailPreferences
       )(implicit hc: HeaderCarrier
       ): Future[Set[String]] = {
-      emailPreferencesService.fetchAPIDetails(applicationSubscriptions.map(_.serviceName).toSet) map { apiDetails =>
+      emailPreferencesService.fetchAPIDetails(applicationSubscriptions.map(_.serviceName.value).toSet) map { apiDetails =>
```

Comment:
Can we push the type down into fetch call rather than unpacking ?

Reply:
Fixed